src/AreaChart/AreaSeries/Area.tsx

```diff
@@ -185,6 +210,16 @@ export const Area: FC<Partial<AreaProps>> = ({
       <MotionPath
         {...extras}
         pointerEvents="none"
```

Comment:
Ok nice, will do thanks

Reply:
Oh actually that utils just for styling the cursor right? So it can't disable mouse events like the `pointerEvents="none"` is doing in this case, can it?

Reply:
Oh duh - IDK what i was thinking lol